docs/content/usage/scenarios/custom-scenario/attack.md

```diff
@@ -13,12 +13,10 @@ The Infection Monkey can simulate various [ATT&CK](https://attack.mitre.org/matr
 
 ## Configuration
 
-- **ATT&CK matrix** You can use the ATT&CK configuration section to select which techniques you want the Infection Monkey to simulate.
-For the full simulation, use the default settings.
+- **Monkey -> Post breach actions** Enable/Disable post breach action in order to trigger an ATT&CK technique on the network. E.g. Trap command triggers Trap ATT&CK technique.
```

Comment:
Since we're moving away from assessment, maybe it's best to just remove the whole scenario? @mssalvatore

Reply:
But the user really needs to know how the mitre attack  techniques can be enabled even without the configuration attack screen.

Reply:
Let's leave it for now. We're going to reimagine our relationship with MITRE ATT&CK, but removing this now is premature.